lib/modules/hosts/youtube.js

```diff
@@ -37,21 +30,16 @@ export default {
 				});
 			} else {
 				// support direct timestamp e.g. t=200
-				starttime = parseInt(starttimeResult[1], 10);
+				starttime = parseInt(params.t, 10);
 				if (isNaN(starttime)) starttime = 0;
 			}
-		}
-
-		let src = `https://www.youtube.com/embed/${hash}?enablejsapi=1&enablecastapi=1&start=${starttime}`;
-
-		if (endtimeResult) {
-			// `&version=3` seems to be required for end time to work
-			src += `&end=${endtimeResult[1]}&version=3`;
-		}
-
-		if (playlistResult) {
-			// appends playlist id if found
-			src += `&list=${playlistResult[1]}`;
+			src = `${src}&start='${starttime}`;
```

Comment:
`+=` was fine here; since the RHS is a string you're guaranteed to get a string (and IMO it's clear enough to read; if we were appending two vars alone then it would be clearer to do ``` `${foo}${bar}` ```)